### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: test/SFSDKInfoTestSuite.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ SDKInfoTestSuite.prototype.testGetInfo = function()  {
             // sdkVersion
         	QUnit.ok(sdkInfo.sdkVersion.indexOf("4.1") == 0, "expected different sdk version");
             // appName
-            QUnit.ok(sdkInfo.appName == "SalesforceHybridSDKTestApp" || sdkInfo.appName == "ForcePluginsTest", "expected different app name");
+            QUnit.ok(sdkInfo.appName == "SalesforceHybridSDKTestApp" || sdkInfo.appName == "SalesforceHybridTest", "expected different app name");
```
Comment (wmathurin) [inline test/SFSDKInfoTestSuite.js]: Hybrid test app is now (since 4.0) called SalesforceHybridTest

File: test/SFSmartStoreTestSuite.js [hunk 2]
```diff
@@ -959,7 +959,7 @@ SmartStoreTestSuite.prototype.testFullTextSearchAgainstArrayNode  = function() {
             return self.upsertSoupEntries(soupName,rawEntries);
         })
         .pipe(function(entries) {
-            var querySpec = navigator.smartstore.buildMatchQuerySpec("attributes.color", "grey");
+            var querySpec = navigator.smartstore.buildMatchQuerySpec("attributes.color", "grey", "descending", 10, "name");
```
Comment (wmathurin) [inline test/SFSmartStoreTestSuite.js]: If we don't specify the order path, it will sort by the query path - in this case a non-leaf node.

**Comment by bhariharan** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

**Comment by khawkins** [comment]
:+1:

**Pull request merged**